### PR TITLE
Replicated dotnet folder

### PR DESCRIPTION
**Pull request by ShreyasRmsft**

### CODE REVIEW

File: dotnet/aspnet/mssqldb/Application/SampleWebApplication.sln [hunk 1]
```diff
@@ -7,10 +7,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SampleWebApplication", "Sam
 EndProject
 Project("{00D1A9C2-B5F0-4AF3-8072-F6C62B433612}") = "Database1", "Database1\Database1.sqlproj", "{9D42710E-7C1C-49F5-B8D2-146844A8B2F2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SampleWebApplication.UnitTests", "SampleWebApplication.UnitTests\SampleWebApplication.UnitTests.csproj", "{F34965FD-E751-4F15-A5D0-759436E0C1B8}"
```
Comment (bishal-pdMSFT) [inline dotnet/aspnet/mssqldb/Application/SampleWebApplication.sln]: remove the files in these projects as well

Comment (ShreyasRmsft) [reply dotnet/aspnet/mssqldb/Application/SampleWebApplication.sln]: they are showing up as moved

Comment (ShreyasRmsft) [reply dotnet/aspnet/mssqldb/Application/SampleWebApplication.sln]: they have been removed

File: dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.UnitTests/SampleWebApplication.UnitTests.csproj [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SampleWebApplication.UnitTests</RootNamespace>
     <AssemblyName>SampleWebApplication.UnitTests</AssemblyName>
-    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```
Comment (bishal-pdMSFT) [inline dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.UnitTests/SampleWebApplication.UnitTests.csproj]: is this change intended?

Comment (ShreyasRmsft) [reply dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.UnitTests/SampleWebApplication.UnitTests.csproj]: nope
reverting
